docs/WindowsBuild.md

```diff
@@ -8,20 +8,10 @@ Visual Studio 2017 or newer is needed to build swift on Windows.
   Development|Universal Windows App Development|Windows SDK" in your
   installation.
 
-## 2. Clone the repositories
-1. Configure git to work with Unix file endings (ie `git config --global core.autocrlf false`)
-1. Create a folder to contain all the Swift repositories
-1. Clone `apple/swift-cmark` into a folder named `cmark`
-1. Clone `apple/swift-clang` into a folder named `clang`
-1. Clone `apple/swift-llvm` into a folder named `llvm`
-1. Clone `apple/swift-compiler-rt` into a folder named `compiler-rt`
-1. Clone `apple/swift` into a folder named `swift`
-1. Clone `apple/swift-corelibs-libdispatch` into a folder named `swift-corelibs-libdispatch`
-1. Clone `apple/swift-corelibs-foundation` into a folder named `swift-corelibs-foundation`
-1. Clone `apple/swift-corelibs-xctest` into a folder name `swift-corelibs-xctest`
-1. Clone `apple/swift-lldb` into a folder named `lldb`
-1. Clone `apple/swift-llbuild` into a folder named `llbuild`
-1. Clone `apple/swift-package-manager` info a folder named `swift-package-manager`
+## 1. Clone the repositories
```

Comment:
Nit: Does it make sense to number the headers like this?  Most of them appear to be 1 now...

Reply:
Yeah, I was hoping that markdown would be smart and renumber them.  I think that I will replace them and remove the numbering in a subsequent change.